aiohttp/web_ws.py

```diff
@@ -179,7 +179,6 @@ def _handshake(self, request: BaseRequest) -> Tuple['CIMultiDict[str]',
         response_headers = CIMultiDict(  # type: ignore
             {hdrs.UPGRADE: 'websocket',
              hdrs.CONNECTION: 'upgrade',
-             hdrs.TRANSFER_ENCODING: 'chunked',
```

Comment:
I wonder if this line removal is everything that you have to do.
Is `web_response.py` changing really necessary?

Reply:
If the answer is *yes* please elaborate.

Reply:
Yes it is required.

The code between [web_response.py#L374-L392](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/web_response.py#L374-L392) always sets `headers[hdrs.TRANSFER_ENCODING] = 'chunked'`.
So the Transfer Encoding Header set in [web_ws.py#L182](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/web_ws.py#L182) does not matter and can be removed.

Interestingly enough `self._chunked` is `False` in [web_response.py#L374](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/web_response.py#L374).

Reply:
Looks like a problem.
I don't want to have these tricks in `web_response.py`. Ideally only `web_ws.py` / `http_websocket.py` should be affected.

Reply:
I reverted `web_response.py` and just overwrite `_length_check` in `web_ws.py` that way we never hit any of the conditions which set the transfer encoding `web_response.py`.

Reply:
@asvetlov any thoughts on my updated pr

Reply:
Sorry for the delay.
I think the PR is in good shape but now I'm super busy by preparing to Python 3.8 feature freeze.
Please let me merge the last planned feature to asyncio and return to aiohttp PRs review after that.
I expect to do it at the next week.